markets/perps-market/contracts/modules/AsyncOrderSettlementPythModule.sol

```diff
@@ -61,39 +61,57 @@ contract AsyncOrderSettlementPythModule is
     }
 
     /**
-     * @dev used for settleing an order.
+     * @notice Settles an offchain order
+     * @param price provided by offchain oracle
+     * @param asyncOrder to be validated and settled
+     * @param settlementStrategy used to validate order and calculate settlement reward 
      */
     function _settleOrder(
         uint256 price,
         AsyncOrder.Data storage asyncOrder,
         SettlementStrategy.Data storage settlementStrategy
     ) private {
+        /// @dev runtime stores order settlement data; prevents stack too deep error
         SettleOrderRuntime memory runtime;
+
         runtime.accountId = asyncOrder.request.accountId;
         runtime.marketId = asyncOrder.request.marketId;
-        // check if account is flagged
+        runtime.sizeDelta = asyncOrder.request.sizeDelta;
+
+        // ensure account has not been flagged for liquidation
+        /// @custom:question is this check necessary given validateRequest() calls isEligibleForLiquidation()?
```

Comment:
sent longer response in dm but checkLiquidation checks if already flagged. and isEligibleForLiquidation checks if account has enough margin. let's leave it alone for now and investigate next week in more depth